teuthology/orchestra/daemon/systemd.py

```diff
@@ -167,10 +167,9 @@ def running(self):
         :return: The PID if remote run command value is set, False otherwise.
         """
         pid = self.pid
-        if pid > 0:
-            return pid
-        else:
+        if pid is None:
```

Comment:
cant we just `return pid` here instead of checking with `if`

Reply:
@rakeshgm  updated the changes.